pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -245,17 +246,18 @@ def unshare_mounts():
             errno = get_errno_loc()[0]
             raise OSError(errno, os.strerror(errno))
 
+    subprocess.check_call(["mount", "--make-rprivate", "/"])
+
 
 def cmd_do(uuid, cmd):
     debug(f"DO {uuid} {cmd}")
     filesystems = list_filesystems()
     for fs in filesystems.values():
         if fs['uuid'] == uuid:
-            path = "/run/cockpit/btrfs"
+            path = os.path.join(TMP_MP_DIR, uuid)
```

Comment:
Is it actually necessary to add the UUID here? Just mounting on /run/cockpit/btrfs should be fine, since it is a private mount. Even mounting on /run/cockpit would be fine, no?

Reply:
I suppose without UUID is fine, I'll give that a try when I get back to this (I have a few other things to fix first).

Reply:
This was done in #21517 , this needs rebasing now.